Exiled.API/Features/Intercom.cs

```diff
@@ -0,0 +1,59 @@
+// -----------------------------------------------------------------------
+// <copyright file="Intercom.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features
+{
+    /// <summary>
+    /// A set of tools to easily handle the Intercom.
+    /// </summary>
+    public static class Intercom
+    {
+        /// <summary>
+        /// Gets or sets the text displayed on the intercom screen.
+        /// </summary>
+        public static string DisplayText
+        {
+            get => global::Intercom.host.CustomContent;
+            set => global::Intercom.host.CustomContent = value;
+        }
+
+        /// <summary>
+        /// Gets or sets the current state of the intercom.
+        /// </summary>
+        public static global::Intercom.State State
+        {
+            get => global::Intercom.host.IntercomState;
+            set => global::Intercom.host.IntercomState = value;
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether or not the intercom is currently being used.
+        /// </summary>
+        public static bool InUse => State == global::Intercom.State.Transmitting || State == global::Intercom.State.TransmittingBypass || State == global::Intercom.State.AdminSpeaking;
+
+        /// <summary>
+        /// Gets the <see cref="Player"/> that is using the intercom. Will be <see langword="null"/> if <see cref="InUse"/> is <see langword="false"/>.
+        /// </summary>
+        public static Player Speaker => !InUse ? null : Player.Get(global::Intercom.host.speaker);
```

Comment:
You can use Player.Get as it will return null if its null

Reply:
> You can use Player.Get as it will return null if its null

Same as above

Reply:
No? Then it has 2 null checks

Reply:
Using just Player.Get(host.speaker) will have a null check on the speaker hub
Then if it's not null, it still still use a Dictionary.TryGet() which is another null check

If we check InUse first and it's false, we can just return null without the null check and enumeration involved with Player.Get()

I'm also not trusting that NW actually clears host.speaker when someone stops speaking.